src/components/Home/Home.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import {
+  TopContainerStyles,
+  MainContainer,
+  RoleOptionContainer,
+  Container,
+  AccountContainer,
+  HeaderContainer,
+  ImageContainer,
+  JoinUsHeader,
+  FooterContainer,
+} from './HomeStyles'
+import { Header } from '../Header'
+import { loggedIn } from '../HeaderNavLink/HeaderNavLink'
+import { PageFooter, Props } from '../PageFooter/PageFooter'
+import { DescriptionOffer } from '../DescriptionOffer/descriptionOffer'
+import { Option } from '../RoleSelectOption/RoleSelectOption'
+import userIcon from '../../stories/assets/user.svg'
+import userIconHovered from '../../stories/assets/userNotHovered.svg'
+import briefcaseIcon from '../../stories/assets/briefcase.svg'
+import briefcaseIconHovered from '../../stories/assets/briefcaseNotHovered.svg'
+import { Login } from '../Login/Login'
+import { LeftSide } from '../LeftSide/LeftSide'
+
+export const Home: React.FC<Props> = ({ footerLinks }) => {
```

Comment:
Create a new folder Pages and put this component inside. Also rename it to HomePage